src/test-suites/null-keys/a.subgraph.ts

```diff
@@ -0,0 +1,29 @@
+import { createSubgraph } from "../../subgraph.js";
+import { books } from "./data.js";
+
+export default createSubgraph("a", {
+    typeDefs: /* GraphQL */ `
+    extend schema
+      @link(
+        url: "https://specs.apollo.dev/federation/v2.3"
+        import: ["@key"]
+      )
+
+    type Query {
+        bookContainers: [BookContainer]
+    }
+    type BookContainer {
+        book: Book
+    }
+    type Book @key(fields: "upc") {
```

Comment:
Can you implement a resolver for it? I know it's useless, but if a gateway calls it for some reason, a missing resolver will cause an error and affect the result.

Reply:
Done 👍